docs/faqs/max_execution_time.rst

```diff
@@ -1,74 +1,85 @@
 .. _faqs/max_execution_time:
 
-============================================================
-What is the maximum execution time for a Qiskit Runtime job?
-============================================================
+=======================================================================
+What is the maximum execution time for a Qiskit Runtime job or session?
```

Comment:
We used to talk about system limits in this section.  Is there a reason that that was removed?

Reply:
System limits (wall clock) was used for both session max time and job max time. But it's no longer used for job max time as that's now quantum time. So I moved it to under session time limit section (and removed the simulator section, as session doesn't apply to simulators).